pkg/controllers/management/auth/prtb_handler.go

```diff
@@ -146,3 +157,59 @@ func (p *prtbLifecycle) reconcileBindings(binding *v3.ProjectRoleTemplateBinding
 	}
 	return p.mgr.grantManagementPlanePrivileges(binding.RoleTemplateName, projectManagmentPlaneResources, subject, binding)
 }
+
+func (p *prtbLifecycle) updateMCAppAndGlobalDNSMembers(prtb *v3.ProjectRoleTemplateBinding) error {
+	mcapps, err := p.mcappLister.List(namespace.GlobalNamespace, labels.NewSelector())
+	if err != nil {
+		return err
+	}
+	if len(mcapps) > 0 {
+		for _, mcapp := range mcapps {
+			updatedMembers := getUpdatedMembers(mcapp.Spec.Members, prtb)
```

Comment:
enqueue all multiclusterapps, and then check if prtb exists in the rbac controller for mcapp and gdns